src/sass_context.cpp

```diff
@@ -112,7 +112,7 @@ extern "C" {
       c_ctx->error_file = sass_strdup(e.pstate.path);
       c_ctx->error_line = e.pstate.line+1;
       c_ctx->error_column = e.pstate.column+1;
-      c_ctx->error_src = e.pstate.src;
+      c_ctx->error_src = sass_strdup(e.pstate.src);
```

Comment:
Sure meanwhile I am reverting it back.

> Did you check if it's the only property affected by this?

I am make the SassError object on libsass.net which captures everything that libsass API has to offer. I made code errors in Sass and hit couple of different exceptions, but this seems to be the only one which was returning garbage.